package.json

```diff
@@ -100,9 +99,10 @@
     "rollup-plugin-delete": "2.1.0",
     "rollup-plugin-typescript2": "0.36.0",
     "storybook": "7.6.20",
-    "styled-components": "5.3.11",
+    "styled-components": "6.1.13",
     "stylelint": "16.10.0",
     "stylelint-order": "6.0.4",
+    "stylis": "4.3.4",
```

Comment:
The migration [docs](https://styled-components.com/docs/faqs#what-do-i-need-to-do-to-migrate-to-v6:~:text=npm%20install%20styled%2Dcomponents%40%5E6.0.0%20stylis%40%5E4.0.0) are misleading. 🤔 `stylis` is a [dependency](https://github.com/styled-components/styled-components/blob/main/packages/styled-components/package.json#L61) of `styled-component`. Therefore, this can be removed. Good catch. 💯